docs/docs/cmd/spo/page/page-section-add.mdx

```diff
@@ -26,10 +26,46 @@ m365 spo page section add [options]
 : Order of the section to add.
 
 `--zoneEmphasis [zoneEmphasis]`
-: Section background shading. Allowed values `None`, `Neutral`, `Soft`, `Strong`
+: Section background shading. Allowed values `None`, `Neutral`, `Soft`, `Strong`, `Image`,`Gradient`
 
 `--isLayoutReflowOnTop`
 : The position of the Vertical section for smaller screens. Applied only for Vertical section.
+
+`--isCollapsibleSection`
+: Set section to be collapsible.
+
+`--showDivider`
+: Show divider line between sections.
+
+`--iconAlignment [iconAlignment]`
+: Expand/collapse icon alignment.
+
+`--isExpanded`
+: Set default display of collapsible section.
+
+`--gradientText [gradientText]`
+: Set gradient setting of a background of a section.
+
```

Comment:
Update the description here that these are only required when zoneEmphasis is `Gradient`

Reply:
the same situation as in previous descriptions